### DISCUSSION TITLE
Is there any guidance about using ltsp to configure a client system for booting from NBD server?

### DISCUSSION THREAD

**Discussion by 593749519** [ltsp/ltsp#805] [General]
as the title implies, just think NBD protocol is simple and easy to port

**Comment by alkisg** [comment]
Switching from NFS to NBD is indeed very simple.
But I'm not aware of any documentation about it.
Personally I'm not interested to document it currently, because in the past NBD has proved to be very unstable; that was the main reason for switching to NFS.
So it's up to the community members to document it if they wish.

**Reply by 593749519** [reply]
Appreciated! I'm just working on a windows NBD server and has it run and want to explore it further
